UI_UX/Frontend/src/stories/index.js

```diff
@@ -38,7 +39,27 @@ storiesOf('Sidebar', module)
     return <AppSidebar clickHandler={action('Clicked nav button')} />;
   });
 
-storiesOf('Header', module).add('view', () => <AppHeader />);
+storiesOf('Header', module)
+  .add('Normal view', () => (
+    <AppHeader
+      loadingBalance={false}
+      exchangeRate={2.13}
+      myBitBalance={215}
+      loadingAccountInfo={false}
+      ethBalance={20}
+      address="0xde0BF ..."
+    />
+  ))
+  .add('Loading', () => (
+    <AppHeader
```

Comment:
React has by default assigned values, so you can just do:
```
<AppHeader
  loadingBalance
  loadingAccountInfo
/>
```
and that should be it. Everything else should be undefined in the component.